datahub/helm/datahub/values.yaml

```diff
@@ -63,8 +72,24 @@ datahub:
     serviceMonitor:
       create: true
     extraInitContainers:
+    - name: wait-for-elasticsearch-job
+      image: gcr.io/pluralsh/bitnami/kubectl:1.25.8
+      imagePullPolicy: IfNotPresent
+      command: [ "/bin/sh", "-c", "until kubectl wait --for=condition=complete --timeout=1s -n datahub job/datahub-elasticsearch-setup-job; do echo 'waiting for datahub-elasticsearch-setup-job'; sleep 1; done" ]
```

Comment:
we had a container that wrapped this before that we had used I can't remember the name though